src/main/java/org/terasology/structureTemplates/internal/systems/StructureSpawnServerSystem.java

```diff
@@ -126,8 +127,11 @@ public void onSpawnStructureWithNoAnimation(SpawnStructureEvent event,
 
         for (RegionToFill regionToFill : spawnBlockRegionsComponent.regionsToFill) {
             Block block = regionToFill.blockType;
-
-            Region3i region = regionToFill.region;
+            if(entity.hasComponent(IgnoreAirBlocksComponent.class)&&block.getURI().getBlockFamilyDefinitionUrn().equals(BlockManager.AIR_ID.getBlockFamilyDefinitionUrn()))
+            {
+                continue;
```

Comment:
I'm not a huge fan of using `continue`... as this is right in the beginning of the code block it acts like a guard or precondition, so I guess using it here is fine..

Please try to follow the overall style guide (not sure what exactly we are using, but this could a be a good orientation: https://google.github.io/styleguide/javaguide.html). Most Java style guides put the opening `{` on the same line 😉